src/components/CompareResults/RevisionHeader.tsx

```diff
@@ -101,6 +101,7 @@ function RevisionHeader(props: RevisionHeaderProps) {
         <Link
           href={getTreeherderURL(header.new_rev, header.new_repo)}
           target='_blank'
+          aria-label={`link to revision jobs in treeherder for ${shortHash}`}
```

Comment:
A tooltip could be useful here (instead of the aria-label) for also sighted users.